test/integration/puppetlabs/general_puppet/general_puppet_int_test.clj

```diff
@@ -154,3 +154,18 @@
         (is (= 500 (:status response)))
         (is (re-matches #".*Cannot retrieve code content because the \"versioned-code\.code-content-command\" setting is not present in configuration.*"
                         (:body response))))))))
+
+(deftest ^:integration static-file-content-works-with-legacy-auth
```

Comment:
Might be better to have this test live in the `auth_conf_test.clj` test namespace?